script/scaffold/templates/config_flow/tests/test_config_flow.py

```diff
@@ -18,8 +18,8 @@
     assert result["errors"] == {}
 
     with patch(
```

Comment:
I think we should prefer to use `patch` imported from `asynctest` generally. It supports both coroutines and normal functions.

Python 3.8 supports patching coroutines, so when that is the minimum we can import patch from unittest.mock again.

We should consider `mock_coro` legacy.

Reply:
I didn't know ! Will migrate.

Reply:
Sorry, accidentally force pushed. Migrated to use asynctest.